mne/decoding/csp.py

```diff
@@ -496,9 +495,9 @@ def plot_filters(self, info, components=None, ch_type=None, layout=None,
             a circle will be used (via .add_artist). Defaults to True.
         colorbar : bool
             Plot a colorbar.
-        scale : dict | float | None
-            Scale the data for plotting. If None, defaults to 1e6 for eeg, 1e13
-            for grad and 1e15 for mag.
+        scalings : dict | float | None
+            The scalings of the channel types to be applied for plotting.
+            If None, defaults to ``dict(eeg=1e6, grad=1e13, mag=1e15)``.
```

Comment:
don't we need to add a `scale: ... DEPRECATED` bit in the docstring?

Reply:
No we usually don't list deprecated params